src/Rings/mpoly-affine-algebras.jl

```diff
@@ -153,9 +153,6 @@ function monomial_basis(A::MPolyQuoRing)
   is_finite_dimensional_vector_space(A) || throw(InfiniteDimensionError())
   I = A.I
   G = standard_basis(I)
-  if dim(I) == -inf # I is the whole ring
```

Comment:
Why remove this? We could transform it to a one-liner, but I don't see why it shouldn't be there.

Reply:
This looks good now.